lib/utils/webpack.config.js

```diff
@@ -262,20 +264,43 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
       loader: 'file-loader',
     })
 
+    const cssModulesConf = 'css?modules&importLoaders=1'
+    const cssModulesConfDev =
+      `${cssModulesConf}&sourceMap&localIdentName=[name]---[local]---[hash:base64:5]`
```

Comment:
Exactly, by default class names are rewritten to hashes, but for development it is more convenient to have extra `[name]---[local]` prefix, so you can easily identify module where class comes from.